tasks/buildsources.js

```diff
@@ -107,6 +107,15 @@ module.exports = function(grunt) {
 				config.rawSource = '\n// '+featureName + '\n' + config.rawSource;
 
 				var featurePath = path.join(destFolder, featureName+'.json');
+                var featureDir = path.dirname(featurePath);
+                mkdir(featureDir, function(err) {
```

Comment:
Ah, it's good as it is then. I assumed async as a callback was being passed in to the function.